.gitignore

```diff
@@ -179,3 +179,6 @@ $RECYCLE.BIN/
 
 !gh-actions/**/dist
 !gh-actions/github/mutex/lib
+
+# Bazel
+MODULE.bazel.lock
```

Comment:
is the lockfile not checked in to repo?

Reply:
hmm - i thought the utilitiy was that it contains the hashes (ie ensures you are downloading exactly the same file/s)

i seem to remember a similar issue with python lockfiles (which are generally checked in)

Reply:
yeah the docs seem to suggest this _should_ be checked in afaict (https://bazel.build/external/lockfile#merge-conflicts)

i think we will need to do this - we cant just ensure trust/reproducibility from just versions